library/swift/EngineBuilder.swift

```diff
@@ -400,6 +425,7 @@ open class EngineBuilder: NSObject {
       dnsFailureRefreshSecondsBase: self.dnsFailureRefreshSecondsBase,
       dnsFailureRefreshSecondsMax: self.dnsFailureRefreshSecondsMax,
       dnsQueryTimeoutSeconds: self.dnsQueryTimeoutSeconds,
+      dnsMinRefreshSeconds: self.dnsMinRefreshSeconds,
```

Comment:
I just realized that we have `dnsFailureRefreshSecondsMax` which puts word `max` as suffix 🤷 may be worth following up to make naming consistent

Reply:
For this PR, the name corresponds exactly with the internal setting. I have thought about going through and cleaning up  some of the inconsistencies in our config API. Perhaps if we create the advanced config split, that would be a good opportunity to also perform some of this cleanup.